main.tex

```diff
@@ -1114,7 +1114,7 @@ \section{Existence of a solution}
 eliminate $h_n, k_m, h_i, k_j$ yb means of the normalizing equations. In this
 form, the complete system is as follows:
 
-\mathbf{Normalizing Equations}
+$\mathbf{Normalizing Equations}$
```

Comment:
```suggestion
\textbf{Normalizing Equations}
```

Reply:
Good catch! Note that `\textbf` is the canonical way to make paragraph text bold, so please include my little patch :)